plugins/lookup/conjur_variable.py

```diff
@@ -86,6 +90,9 @@
 from base64 import b64encode
 from netrc import netrc
 from os import environ
+import stat
+from random import choice
+import string
```

Comment:
For `stat` and `string`, I think you can import just the constants/functions you are using instead of the whole package

Reply:
Thanks, done in next commit